label_sync/main.go

```diff
@@ -361,7 +361,11 @@ func loadLabels(gc client, org string, repos []string) (*RepoLabels, error) {
 				repoLabels, err := gc.GetRepoLabels(org, repository)
 				if err != nil {
 					logrus.WithField("org", org).WithField("repo", repository).Error("Failed listing labels for repo")
-					errChan <- err
+					// IMPROBABLE - BEGIN
```

Comment:
I'm not sure that this will fix the whole issue - this particular error will trigger if label_sync can't read the current labels that exist in the repo, but I don't think that this will prevent label_sync from attempting to make changes to any repos, or what it will do if it tries to update/create labels and fails.

Reply:
You are right. Just updated the PR with a different fixing approach.